rllib/algorithms/apex_dqn/tests/test_apex_dqn.py

```diff
@@ -124,6 +124,8 @@ def test_apex_lr_schedule(self):
             )
             .reporting(
                 min_sample_timesteps_per_iteration=10,
+                # Make sure that results contain info on default policy
+                min_train_timesteps_per_iteration=10,
```

Comment:
In L141, we require results to contain info on default policy.

Reply:
we can probably remove l126 then. 

If we need to merge quickly though then not necessary to make the change.

Reply:
Sure! I've tested removing L126 now and it ends up messing with the way we test the LR here. So we should merge as is.